web-registry/src/lib/normalizers/projects/normalizeProjectsWithCreditClass.ts

```diff
@@ -19,13 +23,15 @@ interface Params {
   projects?: ProjectInfo[] | null;
   projectsMetadata?: (AnchoredProjectMetadataBaseLD | undefined)[];
   projectPagesMetadata?: ProjectPageMetadataLD[];
+  projectParties?: Maybe<PartyFieldsFragment | undefined>[];
   classesMetadata?: (CreditClassMetadataLD | undefined)[];
   sanityCreditClassData?: AllCreditClassQuery;
 }
 
 export const normalizeProjectsWithCreditClass = ({
   projectsMetadata,
   projectPagesMetadata,
+  projectParties,
```

Comment:
should we use a more explicit name like programParties or registryParties?

Reply:
`programParties` sounds good to me, will update it.